src/services/api.tsx

```diff
@@ -39,7 +44,109 @@ export const getPatients = async () => {
         if (name[0].given) patient.firstName = name[0].given.join(", ");
         if (name[0].family) patient.lastName = name[0].family;
       }
+      if (identifier) {
+        patient.identifier = identifier
+          .map((e: any) => {
+            return e.value;
+          })
+          .join(", ");
+      }
+
       return patient;
     }
   );
 };
+
+export const getPatientData = (patientId: string) => {
+  /*
+    getPatientData requests for data from Patient resourcce of id patientId
+    return a Patient object.
+  */
+  const getPatientData = async () => {
```

Comment:
you don't need to have a sub-function, getPatientData can be async

Reply:
corrected